bundler.d/assets.rb

```diff
@@ -5,7 +5,7 @@
 gem 'patternfly-sass', '~> 2.7'
 gem 'multi-select-rails', '~> 0.9'
 gem 'gridster-rails', '~> 0.5'
-gem 'select2-rails', '~> 3.5'
+gem 'select2-rails', '3.5.10'
```

Comment:
@domcleal 4.x (I tried 4.0.1.1) changes all of the classes so it would mean changing all the select2 css.
If you prefer to pin it only in 1.11 I'll open another pr against 1.11-stable and change the css here.